src/interceptors/interceptor.js

```diff
@@ -1,31 +1,34 @@
+import _ from 'lodash';
 import queryString from 'query-string';
 import pathMatch from 'path-match';
 import parseUrl from 'parse-url';
 
-export const interceptor = (serverRoutes, fakeService) => {
-  const getRoute = (handlers, pathname) => {
+export const interceptor = ({routes, host}, fakeService) => {
+  const getRoute = ({handlers, pathname, fullpath}) => {
     const matchesPathname = path => pathMatch()(path)(pathname);
     const route = _.keys(handlers).find(matchesPathname);
+    const hasHost = _.includes(fullpath, host);
```

Comment:
if `host` is undefined or empty will this work? just asking since we also need to allow the user to not specify `host`